### ISSUE TITLE
Menu Error: checkHideMenu is not defined 

### ISSUE THREAD

**Issue by cflinspach** [Icinga/icingaweb2-module-nagvis#24]
I set `headermenu=0` however we were still getting the following when we opened the module in Icingaweb2:

`Javascript error occured: Uncaught ReferenceError: checkHideMenu is not defined http://nsicinga.ad.ccsd.net/nagvis/frontend/nagvis-js/js/NagVisCompressed.js?v=1.9.2 (4489)`

The error would go away when we selected "Show Nagvis Menu" from the dropdown in Icingaweb2. 

We were able to fix the error by changing lines 7 and 17 in configuration.php from 

```
$section = $this->menuSection(N_('Maps'))
    ->setUrl('nagvis/show/map')
    ->setIcon('globe');
```
and 
```
    $section->add($caption, array(
        'url'           => 'nagvis/show/map',
        'urlParameters' => array('map' => $name),
        'priority'      => ++$prio
    ));
```
to
```
$section = $this->menuSection(N_('Maps'))
    ->setUrl('nagvis/show/map?showMenu=1')
    ->setIcon('globe');
```
and
```
    $section->add($caption, array(
        'url'           => 'nagvis/show/map?showMenu=1',
        'urlParameters' => array('map' => $name),
        'priority'      => ++$prio
    ));
```

**Comment by rahonalab** [comment]
@cflinspach workaround doesn't work on current (git) nagvis module.

any ideas how to fix?

**Comment by bmccorkle** [comment]
Getting a similar error when using an interactive map...

> Javascript error occured:  Uncaught ReferenceError: checkHideMenu is not defined
> https://icinga.domainname.com/nagvis/frontend/nagvis-js/js/ViewWorldMap.js?v=1.9.7 (80)

I used rahonalab's suggestion and commented out the following three lines at line 80 which seems to work so far

```
 // hide eventual open header dropdown menus when clicking on the map
if (checkHideMenu)
   g_map.on('mousedown', checkHideMenu);
g_map.on('mousedown', context_handle_global_mousedown);
```


icingaweb2-module-nagvis 1.1.1
nagvis 1.9.7

**Comment by wernerfred** [comment]
Figured out the same solution, damn that i didn't found this issue here, would have saved a lot of time -.-
I described the problem and my solution here: https://github.com/NagVis/nagvis/issues/162

I think it's important to mention that changing something in ```NagVisCompressed.js``` did not show any effect. Therefore i deleted this file and the Error message changed from ```NagVisCompressed.js``` to ```ViewWorldmap.js``` as mentioned by @bmccorkle.
I then simply deleted the lines 80 to 83 and the error message was completely gone and not only that another problem (infinite loading at top till any action with mouse) was also solved (mentioned here: https://github.com/NagVis/nagvis/issues/160)

I also did not run in other problems or could figure out any functionality which could be broken through my changes. If someone could proof that i would send a pull request :)
